### PR TITLE
Corazone renamed to Higadrite, adds replacement chemical for abductors and fixes heart disease

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request
Renames Corazone to Higadrite, as #47536 made it only stabilize the liver, and Corazone is spanish for heart.
Fixes Miocardial Infarction's flavor text that says Corazone is a cure, and it adding Corazone to your bloodstream even though it no longer stabilizes the heart.
Adds a new reagent, Cordiolis Hepatico. Acts as old Corazone, only injectable ingame with an abductor surgery table.

## Why It's Good For The Game
As a spanish dialect speaker, corazone being named the same hurts my corazon. This will make it clearer that it's different for everyone attempting to make old corazone, and helps understand what it is a bit more clearly if you have a bilingual bonus.
Fixing heart disease not being properly updated is obviously beneficial.
Abductor table had a heart and liver stabilizing chemical being constantly injected into the victim/patient, so now it has that again.

## Changelog
:cl:
tweak: As corazone no longer stabilizes the heart, It's been renamed to Higadrite.
fix: Fixes heart disease telling you to utilize corazone, and injecting you with it during its final stages. It now injects Penthrite.
fix: Abductor surgery table injects a replacement chemical for old Corazone.
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 1]
```diff
@@ -1049,20 +1049,35 @@
...
+	..()
 	REMOVE_TRAIT(M, TRAIT_STABLELIVER, type)
+
+/datum/reagent/medicine/cordiolis_hepatico
+	name = "Cordiolis Hepatico"
+	description = "A strange, pitch-black reagent that seems to absorb all light. Effects unknown."
+	color = "#000000"
+	self_consuming = TRUE
+
+/datum/reagent/medicine/higadrite/on_mob_add(mob/living/M)
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: when you copy+pasted this (and the bit below this), you forgot to replace "higadrite" with "cordiolis_hepatico"

**Comment by ATH1909** [comment]
why the hell did you feel the need to nerf CORAZONE?

NOBODY makes that shit (it's INCREDIBLY niche), and it's only really useful for "look at me, ma, I can survive without internal organs!" memes

with this PR, I'd have to make 500u of both corazone AND penthrite, and penthrite looks like it's gonna be a bitch to make in large quantities (I won't care about the stomach damage, of course, but 

if you're gonna do this, at LEAST make penthrite not deal stomach damage to you. NOBODY's going to use a chem that's EVEN MORE NICHE than corazone is that also HAS A DOWNSIDE

**Comment by ATH1909** [comment]
wait a minute, never mind, the PR that added penthrite also removed the heart-stabilizing effect of corazone

carry on

**Comment by ArcaneMusic** [comment]
Higadrite is a rock name. You are following rock and stone naming convention. I'm not a fan of the existing name scheme for chems, but this DOES spit in the face of the rock scientists.

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm (hunk 1 above)
Comment (carlarctg) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: > Higadrite is a rock name. You are following rock and stone naming convention. I'm not a fan of the existing name scheme for chems, but this DOES spit in the face of the rock scientists.

chems with -rite suffix work by synthesizing into kidney stones with various effects

**Comment by carlarctg** [comment]
> Higadrite is a rock name. You are following rock and stone naming convention. I'm not a fan of the existing name scheme for chems, but this DOES spit in the face of the rock scientists.

chems with -rite suffix work by synthesizing into kidney stones with various effects

**Comment by xurxomario** [comment]
> Higadrite is a rock name. You are following rock and stone naming convention. I'm not a fan of the existing name scheme for chems, but this DOES spit in the face of the rock scientists.

And higado is also the spanish for liver, so it doubly works. I aproove of this renaming big time.

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 2]
```diff
@@ -1049,20 +1049,35 @@
...
 	return TRUE
 
-/datum/reagent/medicine/corazone
-	//Corazone no longer stops heart attacks. look into C2 penthrite.
-	name = "Corazone"
-	description = "A medication used to treat pain, fever, and inflammation, along with failing livers."
-	color = "#F49797"
+/datum/reagent/medicine/higadrite
+	name = "Higadrite"
+	description = "A medication utilized to treat ailing livers."
```
Comment (Angustmeta) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: failing

**Comment by cacogen** [comment]
my eyes just glaze over when i see all this chem bullshit

**Comment by carlarctg** [comment]
DNM for a while please

**Comment by coiax** [comment]
I mean, originally when I added corazone, way back, it was only a "survive without a heart" drug, so the name made sense.

I'm probably only -0 on this because I named it originally.

**Comment by EdgeLordExe** [comment]
merge this please, i actually didnt know that abductor's surgery table used corazone when i created penthrite

**Comment by cacogen** [comment]
can you just name things something where it's self-explanatory what it does like a normal person

**Pull request merged**